src/server/services/StatisticsService.ts

```diff
@@ -20,29 +20,43 @@ export default class StatisticsService {
    * @param event The event to gather statistics for.
    */
   getEventStatistics(event: EventDocument): Promise<EventStatistics> {
-    return Promise.all(
-      [this.UserModel.countDocuments({event: event}),
-        this.UserModel.find({
-          event: event._id,
-        }).distinct('university').exec(),
-        this.UserModel.aggregate([
-          {
-            $match: {event: event._id},
-          },
-          {
-            $group: {_id: '$gender', count: {$sum: 1}},
-          },
-        ]).exec(),
-        this.UserModel.countDocuments({event: event, checkedIn: true}),
-        this.UserModel.aggregate([
-          {
-            $match: {event: event._id},
-          },
-          {
-            $group : {_id: '$status', count: {$sum: 1}},
-          }]).exec(),
-        this.UserModel.countDocuments(getResumeConditions(event)),
+    return Promise.all([
+      this.UserModel.countDocuments({ event: event }),
+      this.UserModel.find({
+        event: event._id
+      })
+        .distinct('university')
+        .exec(),
+      this.UserModel.aggregate([
+        {
+          $match: { event: event._id }
+        },
+        {
+          $group: { _id: '$gender', count: { $sum: 1 } }
+        }
+      ]).exec(),
+      this.UserModel.countDocuments({ event: event, checkedIn: true }),
+      this.UserModel.aggregate([
+        {
+          $match: { event: event._id }
+        },
+        {
+          $group: { _id: '$status', count: { $sum: 1 } }
+        }
+      ]).exec(),
+      this.UserModel.countDocuments(getResumeConditions(event)),
+      this.UserModel.aggregate([
```

Comment:
Nice aggregate!

Reply:
Thanks! I'll address your comments and reviews later this week!